lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8794,23 +8796,42 @@ ClangImporter::Implementation::importSwiftAttrAttributes(Decl *MappedDecl) {
     // FIXME: Hard-code @MainActor and @UIActor, because we don't have a
     // point at which to do name lookup for imported entities.
     if (isMainActorAttr(swiftAttr)) {
-      if (SeenMainActorAttr) {
+      if (seenMainActorAttr) {
         // Cannot add main actor annotation twice. We'll keep the first
         // one and raise a warning about the duplicate.
         HeaderLoc attrLoc(swiftAttr->getLocation());
         diagnose(attrLoc, diag::import_multiple_mainactor_attr,
                  swiftAttr->getAttribute(),
-                 SeenMainActorAttr.getValue()->getAttribute());
+                 seenMainActorAttr.getValue()->getAttribute());
         continue;
       }
 
       if (Type mainActorType = SwiftContext.getMainActorType()) {
         auto typeExpr = TypeExpr::createImplicit(mainActorType, SwiftContext);
         auto attr = CustomAttr::create(SwiftContext, SourceLoc(), typeExpr);
         MappedDecl->getAttrs().add(attr);
-        SeenMainActorAttr = swiftAttr;
+        seenMainActorAttr = swiftAttr;
+      }
+
+      continue;
+    }
+
+    if (isMutabilityAttr(swiftAttr)) {
+      if (seenMutabilityAttr) {
+        const clang::SwiftAttrAttr *nonmutatingAttr;
```

Comment:
Nit: please always initialize pointers with nullptr. Here it's obvious that it's getting initialized, but this code may change in the future and another path might be added where this is not initialized. It's just better to 100% of the time initialize these with nullptr (I've been bitten by this several times before). Alternatively, maybe a ternary operator here? Your choice.

Reply:
Good point, I've rewritten it to a ternary operator 👍🏽